ai-data/generative-apis/how-to/query-text-models.mdx

```diff
@@ -0,0 +1,164 @@
+---
+meta:
+  title: How to query text models
+  description: Learn how to interact with powerful text models using Scaleway's Generative APIs service.
+content:
+  h1: How to query text models
+  paragraph: Learn how to interact with powerful text models using Scaleway's Generative APIs service.
+tags: generative-apis ai-data text-models
+dates:
+  validation: 2024-08-28
+  posted: 2024-08-28
+---
+
+Scaleway's Generative APIs service allows users to interact with powerful text models hosted on the platform.
+
+There are several ways to interact with text models
+- The Scaleway [console](https://console.scaleway.com) will soon provide a complete [playground](/ai-data/generative-apis/how-to/query-text-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real time.
+- Via the [Chat API](/ai-data/generative-apis/how-to/query-text-models/#querying-text-models-via-api)
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/) (Necessary for API authentication)
+- Python 3.7+ installed on your system.
+
+## Accessing the Playground
+
+Scaleway's Playground is in development, stay tuned!
```

Comment:
Maybe link to the changelog (filtered with the product category) so clients can refer to it to check for updates?

Reply:
In private beta status, we rarely publish any changelog entry